pilot/pkg/model/push_context.go

```diff
@@ -94,7 +94,7 @@ type PushContext struct {
 
 	// AuthzPolicies stores the existing authorization policies in the cluster. Could be nil if there
 	// are no authorization policies in the cluster.
-	AuthzPolicies *AuthorizationPolicies `json:"-"`
+	AuthzPolicies *AuthorizationPolicies `json:"authz_policies"`
```

Comment:
This should be reverted

Reply:
See https://github.com/istio/istio/pull/18870

Reply:
I'll push a commit to this PR to remove the annotation and send another PR for master, is this all we need?

Reply:
If #18870 merges then we only need to change this on 1.4 back to `-`. Otherwise also on master.

The problem is this gets logged and outputs a giant message when we only mean to log errors

Reply:
We don't need the annotation actually, just pushed a commit to remove it, will send another PR for master.